pkg/api/handlers.go

```diff
@@ -211,7 +212,11 @@ func NewQuerierHandler(
 		false,
 		regexp.MustCompile(".*"),
 		func() (v1.RuntimeInfo, error) { return v1.RuntimeInfo{}, errors.New("not implemented") },
-		&v1.PrometheusVersion{},
+		&v1.PrometheusVersion{
```

Comment:
Why we decided not to support the following?

```
BuildUser string `json:"buildUser"`
BuildDate string `json:"buildDate"`
GoVersion string `json:"goVersion"`
```

Not that I want to support them, but I thought it do be nice to have explicit discussion on this as documentation purposes.

Reply:
I could add GoVersion for sure. Thanks for catching.
But for other variables, right now we don't propagate these variables right now from our build.